css-positioning/absolute.css

```diff
@@ -0,0 +1,40 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
```

Comment:
you need code here to make sure the overlay element is removed from the document flow and it has the same height and width as its parent to darken the image

Reply:
<img width="1023" alt="absolute-positioning" src="https://user-images.githubusercontent.com/78944582/112383559-7f1f0380-8caa-11eb-9b26-d19a8a18fff1.png">